webpack.config.js

```diff
@@ -3,4 +3,14 @@ const path = require('path')
 
 webpackConfig.entry.dashboard = path.join(__dirname, 'src', 'dashboard.js')
 
+webpackConfig.optimization = {
+	splitChunks: {
+		cacheGroups: {
+			defaultVendors: {
+				name: 'commons',
```

Comment:
```suggestion
				name: 'dashboard',
```
This fixed it, at least in my context. Was it a mistake or did you intentionally wrote "commons" here?

Reply:
Looking a bit at the Webpack doc and other apps, maybe this is the proper fix:
```
__webpack_nonce__ = btoa(OC.requestToken) // eslint-disable-line
__webpack_public_path__ = OC.linkTo('notes', 'js/') // eslint-disable-line
```

I don't quite get how one chooses what to put in which chunk file.

Reply:
Actually the second one would be the proper fix, sorry I missed that.